vitals/StateEstimator.java

```diff
@@ -0,0 +1,63 @@
+package vitals;
+
+public class StateEstimator {
```

Comment:
StateEstimator - very generic name ...refactor

Reply:
Refactored StateEstimator  to CheckAndAlertBatteryFactors.